packages/components/Button/src/CompoundButton/SPEC.md

```diff
@@ -4,6 +4,8 @@
 
 The `CompoundButton` is a `Button` that is designed to show an extra line of text.
 
+Does not render on Android.
```

Comment:
nits: remove the extra line above. If not already, run prettier across the change